ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -446,6 +458,33 @@ internal sealed class ArchiWebHandler : IDisposable {
 			return await UrlGetToHtmlDocumentWithSession(SteamCommunityURL, request).ConfigureAwait(false);
 		}
 
+		internal async Task<HashSet<ulong>> GetDigitalGiftCards() {
+			const string requestGifts = SteamStoreURL + "/gifts";
+			HtmlDocument response = (await WebLimitRequest(SteamStoreURL, async () => await WebBrowser.UrlGetToHtmlDocument(requestGifts).ConfigureAwait(false)).ConfigureAwait(false))?.Content;
```

Comment:
Do not call `WebBrowser` directly, use `UrlGetToHtmlDocumentWithSession()` instead. This will entirely fail if your session is outdated or bot is disconnected, `UrlGetToHtmlDocumentWithSession()` takes care of that, and web limiting too. See other functions.